cli.js

```diff
@@ -24,19 +25,31 @@ if (!username) {
 	process.exit(1);
 }
 
-const user = await npmUser(username);
-const rows = [];
-
-const createRow = (prefix, key) => {
-	if (user[key]) {
-		rows.push(`${prefix}: ${user[key]}`);
+try {
+	const user = await npmUser(username);
+	const rows = [];
+
+	const createRow = (prefix, key) => {
+		if (user[key]) {
+			rows.push(`${logSymbols.info} ${prefix}: ${user[key]}`);
+		}
+	};
+
+	createRow('Name', 'name');
+	createRow('Email', 'email');
+	createRow('GitHub', 'github');
+	createRow('Twitter', 'twitter');
+
+	console.log(rows.join('\n'));
+} catch (error) {
+	// TODO:
+	// Or: error.response.status === 404
+	// Maybe `npm-user` should throw a new Error with code = 404?
+	if (error.message === 'User doesn\'t exist') {
+		console.error(`${logSymbols.error} User \`${username}\` could not be found.`);
+		process.exit(1);
 	}
```

Comment:
Thoughts? And looking back at it, `User doesn't exist` probably isn't the most descriptive error message 😅

Reply:
I would just improve the error in `npm-user`, so this could simply just rethrow the error with the symbol added. Could maybe also add a `code` property for other use-cases.